scrapi/harvesters/addis_ababa.py

```diff
@@ -6,6 +6,33 @@
 from __future__ import unicode_literals
 
 from scrapi.base import OAIHarvester
+from scrapi.base import helpers
+
+
+def oai_process_uris_addis_ababa(*args):
+
+    identifiers = helpers.gather_identifiers(args)
+    provider_uris, object_uris = helpers.seperate_provider_object_uris(identifiers)
+
+    potential_uris = (provider_uris + object_uris)
+
+    for i, uri in enumerate(potential_uris):
+        potential_uris[i].replace('http://handle.net/123456789/', 'http://etd.aa.edu.et/handle/123456789')
+
+    try:
+        canonical_uri = potential_uris[0]
+    except IndexError:
+        raise ValueError('No Canonical URI was returned for this record.')
+
+    provider_uris = potential_uris[0:len(provider_uris)]
+
+    object_uris = potential_uris[(len(provider_uris) + 1):len(potential_uris)]
```

Comment:
Shouldn't this one just get the rest? It shouldn't need the second part of the slice, unless I am mistaken.